src/core_modules/capture-core/components/Pages/common/TEIRelationshipsWidget/RegisterTei/RegisterTei.component.tsx

```diff
@@ -23,23 +22,35 @@ const getStyles = () => ({
     },
 });
 
-const CardListButton = (({ teiId, values, handleOnClick }) => (
+type CardListButtonProps = {
+    teiId: string;
+    values: Record<string, any>;
+    handleOnClick: (teiId: string, values: Record<string, any>) => void;
+};
+
+const CardListButton = (({ teiId, values, handleOnClick }: CardListButtonProps) => (
     <Button
         small
         dataTest="view-dashboard-button"
         onClick={() => { handleOnClick(teiId, values); }}
     >
-        {i18n.t('Link')}
+        {i18n.t('Link') as string}
     </Button>
 ));
 
-const DialogButtons = ({ onCancel, onSave, trackedEntityName }) => (
+type DialogButtonsProps = {
+    onCancel: () => void;
+    onSave: () => void;
+    trackedEntityName: string | undefined;
+};
+
+const DialogButtons = ({ onCancel, onSave, trackedEntityName }: DialogButtonsProps) => (
     <>
         <Button
             onClick={onCancel}
             secondary
         >
-            {i18n.t('Cancel')}
+            {i18n.t('Cancel') as string}
```

Comment:
You should not have to add a string on all of these. If you do try running `yarn` again and restarting the TypeScript server.